### PR TITLE
Ruby diagnostic response

### PR DESCRIPTION
**Pull request by seatuna**

### CODE REVIEW

File: lib/diagnostic.rb [hunk 1]
```diff
@@ -40,7 +40,9 @@
 def method_name
 end
 
-Diagnostic.defined_method = # call your method here
+Diagnostic.defined_method = def sum_times_two (a, b)
+                              2 * (a + b)
+                            end
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: define then call
```

def add_and_multiply(a, b)
  2 * (a + b)
end

Diagnostic.defined_method = add_and_multiply(3, 4)
```

File: lib/diagnostic.rb [hunk 2]
```diff
@@ -49,31 +51,33 @@ def method_name
 
 # define your method here
 
-Diagnostic.predicate = # call your method here
+Diagnostic.predicate = def odd? (number)
+                          number.odd?
+                      end
 
 # Numbers in Ruby
```
Comment (J-Weeks) [inline lib/diagnostic.rb]: define then call

**Comment by J-Weeks** [comment]
14/16.  Good job!  Keep up the good work.

File: lib/diagnostic.rb (hunk 1 above)
Comment (seatuna) [inline lib/diagnostic.rb]: I see. I just realized I misread the question / maybe wasn't reading!

**Pull request closed without merging**